docs/transform.md

```diff
@@ -9,7 +9,7 @@ transforms:
     my_transform:
         input: $input # (or some actual name)
         output: $output # (or some actual name)
-        stages: 
+        subtransforms: 
```

Comment:
i thought about it more, i think in the config (ONLY in the config) this can be changed to `steps`
```suggestion
        steps: 
```